### DISCUSSION TITLE
correction current injection (CCI) power flow using YALMIP

### DISCUSSION THREAD

**Discussion by Layan523** [yalmip/YALMIP#1315] [Optimization modelling] [answered]
Hi Johan,

I am trying to model a correction current injection (CCI) power flow using YALMIP. In CCI the load is represented by an admittance (Y(n)= S/V(n-1)). where S is the constant power and V(n-1) is the voltage (optimization variable) calculated in the previous iteration. 

is there a way in YALMIP to recover the value of the voltage (V) from the solver to calculate Y(n) and supply it to the solver before every iteration?

Thank you
Layan

**Comment by johanlofberg** [comment, marked as answer]
You mean a callback in the solver to look at the iterations? Not possible (unless you are using a nonlinear solver based on callbacks for function evaluations such as fmincon, for which you can put a break in YALMIPs callback layer such as fmincon_fun and then hack that
